app/utils/general.js

```diff
@@ -69,3 +69,13 @@ export function throttle(fn, limit, ...args) {
         }
     };
 }
+
+export function isEmail(email) {
```

Comment:
taken from webapp utils.

Reply:
We might want to move this into `mattermost-redux` so that both apps can share code for it